data/schema.graphql

```diff
@@ -2499,6 +2560,7 @@ type OrderedSet {
 
 type organizer {
   profile_id: ID
+  profile: Profile
```

Comment:
is "Profile" here the same as what we were talking about the other week? Metadata to an entity?